bin/gwu/gwuLib.ml

```diff
@@ -1308,12 +1308,13 @@ let print_relations opts base gen ml =
   in
   let rec loop = function
     | [] -> ()
-    | (p, if_def) :: pl ->
+    | (p, _if_def) :: pl ->
         let def_p = ref [] in
+        let if_def = Collection.Marker.get gen.mark_rel (Driver.get_iper p) in
```

Comment:
At a quick glance, I think something is odd or I am missing something.
Here you have if_def be the value stored in the marker

but then in print_relations_for_person
https://github.com/geneweb/geneweb/blob/8623eedb65ca75daf33d8c49d5b500d2026d9d63/bin/gwu/gwuLib.ml#L1279

https://github.com/geneweb/geneweb/blob/8623eedb65ca75daf33d8c49d5b500d2026d9d63/bin/gwu/gwuLib.ml#L1285

Both can't be true at the same time

Reply:
That is if the previous if_def was not properly computed, do you have a process that produces a gw with several definitions for the same person in the current version ?

Reply:
my post 4 days ago above gives a 3 person base demonstrating the problem.
I "tried" with gen.mark rather than gen.mark_rel, and it did not solve the issue.
I admit that this is "blind testing"!!